static/css/mozsearch.css

```diff
@@ -777,9 +792,15 @@ span[data-symbols].hovered {
   box-shadow: 3px 3px 3px rgba(0, 0, 0, .5);
   border-radius: 3px;
   text-align: left;
-  z-index: 200;
+  z-index: 101;
   cursor: auto;
 }
+.blame-popup code {
+  background-color: var(--page-background);
+  display: block;
+  padding: 0.5em;
+  white-space: pre;
+}
```

Comment:
I think we need/want `color: var(--page-foreground);` in here too; otherwise for me (using the default/light color theme), identifier tokens are white text on a white background.

Reply:
Note that I vim'ed this change on the web server for the moment.